cmake/Config.cmake

```diff
@@ -87,11 +87,21 @@ if(MSVC)
     endif()
 elseif(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
     set(SFML_COMPILER_CLANG 1)
+
+    execute_process(COMMAND "${CMAKE_CXX_COMPILER}" "-v" ERROR_VARIABLE CLANG_COMPILER_VERSION OUTPUT_VARIABLE CLANG_COMPILER_VERSION)
```

Comment:
Nitpick: I'd order it the other way around, first `OUTPUT_VARIABLE` then `ERROR_VARIBLE`